vignettes/manual/AddNewComponent.Rmd

```diff
@@ -7,15 +7,16 @@ Step one:
 -----------
 
 Add two new files:  
-* headers/core/X.hpp
-* source/core/X.cpp
+* `headers/core/X.hpp`
+* `source/core/X.cpp`
 
 It is best to copy and modify the corresponding files from an already constructed component.
 
 Step two (within X.hpp):  
 -----------
-Under `private` list the variables that are used in the component and their corresponding type. E.g. 
-* `tseries<unitval> x` to create a [time series](TSeries.html).
+Under `private` list the variables that are used in the component and their corresponding type. 
+E.g. 
+* `tseries<unitval> x` to create a [time series](https://jgcri.github.io/hector/articles/manual/TimeSeries.html).
```

Comment:
Maybe add: "Because of Hector's ability to \[reset](link), every state variable generally needs an associated time series history variable."